frontend/src/layouts/MainLayout.js

```diff
@@ -72,30 +48,19 @@ const useStyles = makeStyles(theme => ({
     }),
     marginLeft: 0,
   },
+  navbarOffset: {
+    ...theme.mixins.toolbar,
+    "@media (max-width: 700px)": {
+      minHeight: "116px",
+    },
+  },
```

Comment:
I added an offset element so that the fixed header doesn't overlap the top of the page.